src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WithRepresentativePersonalisationTest.java

```diff
@@ -103,6 +110,7 @@ private String getFieldValue(NotificationEventType notificationEventType, String
         return new Object[]{
             new Object[]{APPEAL_LAPSED_NOTIFICATION},
             new Object[]{APPEAL_WITHDRAWN_NOTIFICATION},
```

Comment:
We don't sent notification for rep while creating the appeal, I think some gaps on this jira